### DISCUSSION TITLE
Improvement of Pareto front  for Multi-objective BO qEHVI

### DISCUSSION THREAD

**Discussion by Yoon-SuKyung** [meta-pytorch/botorch#2218] [Q&A]
I am working with the aim of optimizing the airfoil geometry.
Objective1 is the lift coefficient, and objective2 is the drag coefficient. Since all of them need to be maximized, 1-y was applied to objective2.

The purple dot in the figure is the sample point obtained through DOE and the result was obtained.
My expectation is that qEHVI, qParEGO points will be located in a place similar to the pareto-front shown in the figure, but it is not. Why is that? Can't we improve the pareeto front?
so, is it possible to obtain a pareto front for a wider area?

![figure](https://github.com/pytorch/botorch/assets/98367263/97aebb3c-624a-41ef-a513-191ed647288e)

Attached is my code.

[code_ver1.txt](https://github.com/pytorch/botorch/files/14404536/code_ver1.txt)

[Re05000_M02.txt](https://github.com/pytorch/botorch/files/14404518/Re05000_M02.txt)

**Comment by sdaulton** [comment]
I see the reference point is set to [0.0,0.0], but the 1-y transform is applied to the second objective---so the transformed second objective would have values between ~ 0.8 and 0.95. Since the problem is then to maximize both objectives, the pareto frontier would be very far from the reference point [0,0]. I'd recommend adjusting the reference point to be slightly worse than the pareto frontier (in the transformed space)---e.g. something like [0.0, 0.75] or better yet use the built-in reference point heuristic: https://github.com/pytorch/botorch/blob/0e74bb60be3492590ea88d6373d89a877c6a52c1/botorch/utils/multi_objective/hypervolume.py#L65
